create-toolchain

```diff
@@ -106,11 +112,26 @@ cp -a "${mac_toolchain}"/usr/include linux/iphone/include
 # ...but-but replace the mac lib/swift/clang symlink with the linux one
 rm linux/iphone/lib/swift/clang
 cp -a linux/{host,iphone}/lib/swift/clang
+# remove borked plutil in favor of plistutil
+rm linux/iphone/bin/plutil
+# remove c++ headers as included in SDK
+rm -rf linux/iphone/include/c++/
 
 info "Installing Darwin tools"
 # The hierarchy lines up with ours just right so we dont need to -C
-dl "darwin-tools-${linux_version}.tar.gz" "https://github.com/kabiroberai/darwin-tools-linux/releases/download/v${DARWIN_TOOLS_VERSION}/darwin-tools-${linux_version}.tar.gz"
-tar -xzvf "darwin-tools-${linux_version}.tar.gz"
+# If distro isn't debian-based, use ubuntu20.04 as darwintools base
+if ! [[ "${linux_version}" == *"ubuntu"* ]]; then
```

Comment:
readability nit: let's invert this statement to `if [[ "${linux_version}" == *"ubuntu"* ]]; then` and swap the branches around.